db/seeds.rb

```diff
@@ -92,7 +92,7 @@
                         brand: 'Ride',
                         image: File.open('app/assets/images/machete_gt.png')
                       },
-                      { name: 'WILD LIFE WIDE SNOWBOARD',
+                      { name: 'WILD LIFE WIDE',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.